tests/test_logging_config.py

```diff
@@ -97,6 +93,33 @@
 SETTINGS_DEFAULT_NAME = 'custom_airflow_local_settings'
 
 
+def reset_logging():
+    """Reset Logging"""
+    manager = logging.root.manager
+    manager.disabled = logging.NOTSET
+    airflow_loggers = [
+        logger for logger_name, logger in manager.loggerDict.items() if logger_name.startswith('airflow')
+    ]
+    for logger in airflow_loggers:  # pylint: disable=too-many-nested-blocks
+        if isinstance(logger, logging.Logger):
```

Comment:
could we not just reload the logging config?

Reply:
We already do that in `settings_context` but it does seem to clear/reset everything:

https://github.com/apache/airflow/blob/7285e61c42c89799e967ce9f0d391707ee02664d/tests/test_logging_config.py#L186-L187

Reply:
Probably because of https://github.com/apache/airflow/blob/7285e61c42c89799e967ce9f0d391707ee02664d/airflow/config_templates/airflow_local_settings.py#L59

But we probably want to keep that. Hm.